models/classes/media/sourceStrategy/HttpSource.php

```diff
@@ -113,13 +115,17 @@ public function getBaseName($link)
         return $realfilename;
     }
 
+    public function getDirectories(DirectorySearchQuery $params): array
+    {
+        return $this->getDirectory($params->getParentLink());
```

Comment:
Not really required the parameter.

```suggestion
        return $this->getDirectory();
```